test/e2e_new/conformance_test.go

```diff
@@ -99,3 +105,56 @@ var _ = Describe("conformance tests", func() {
 	})
 
 })
+
+func prepConformanceConfig(artifactsDir string, e2eConfig *clusterctl.E2EConfig) error {
+	for i, p := range e2eConfig.Providers {
+		if p.Name != "aws" {
+			continue
+		}
+		e2eConfig.Providers[i].Files = append(p.Files, clusterctl.Files{
```

Comment:
we could also leave the copying and kustomize call in the Makefile and then append the file here after ensuring it exists.

Reply:
Idea is to start moving towards not needing the makefile, and allowing `go test` to work if you wanted to use it that way.